code/style.css

```diff
@@ -1,7 +1,145 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+* {
+margin: 0;
+padding: 0;
+box-sizing: border-box;
+-webkit-font-smoothing:antialiased; 
+text-rendering: optimizeLegibility
+
+}
+
+section {
+  padding: 8rem 10rem;
+}
+
+body {
+font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+background-color: cornsilk;
+
+}
+
+.header {
+height: 50vh;
+min-height: 50vh;
+width: 100vw;
+background-image: url("./images/macarons.jpg");
```

Comment:
Nice! I didn't know you could add an image into the css! Learning new things everyday!